CODE_OF_CONDUCT.adoc

```diff
@@ -0,0 +1,142 @@
+== Contributor Covenant Code of Conduct

```

Comment:
In the Markdown version, this was a [h1 heading](https://www.w3schools.com/tags/tag_hn.asp) (Markdown: `#`, AsciiDoc: `=`), "Our Pledge" was a h2 heading (Markdown: `##`, AsciiDoc: `==`), and so on.

Please keep the original formatting in this regard. You can also check out the [corresponding AsciiDoc documentation](https://docs.asciidoctor.org/asciidoc/latest/syntax-quick-reference/#section-titles).